packages/reports/addon/controllers/reports/report.js

```diff
@@ -14,6 +14,11 @@ const REPORT_STATE = {
 };
 
 export default Controller.extend({
+  /**
+   * @property {Boolean} showSaveAs - whether the save as dialog is showing
+   */
+  showSaveAs: false,
```

Comment:
Can we check that this state is reset when loading the view?

Reply:
Done